src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -84,4 +85,12 @@
         @Param("maxFailureCount") int maxFailureCount,
         Pageable pageable
     );
+
+    @Query(
+        nativeQuery = true,
+        value = "SELECT COUNT(1) AS incomplete\n"
+            + "FROM envelopes\n"
+            + "WHERE DATE(createdat) < :date and status != 'COMPLETED'"
```

Comment:
Should it be not just doc notification sent ? else doc upload failure metadata validation failure will be included which we don't want ?

Reply:
let me review the states. i believe i missed this step in my tiny (as of yet) stream of commits